packages/preprocess/index.cjs

```diff
@@ -30,13 +30,12 @@ const getPrismLangs = function(){
 
 const getRouteHash = function(filename){
     let route = filename.split("/src/pages")[1] === "/+page.md" ? "/" : filename.split("/src/pages")[1].replace(".md","").replace(/\/\+page/g,"")
-    let routeHash = md5(route)
-    return routeHash
+    return md5(route)
 }
 
 const createDefaultProps = function(filename, componentDevelopmentMode, fileQueryIds){
     let componentSource = componentDevelopmentMode ? '$lib' : '@evidence-dev/components';
-    let routeHash = getRouteHash(filename)
```

Comment:
routeHash -> routeH?

Reply:
Yes, the same name was used twice as we have an import on `routeHash`